bitnami/jenkins/templates/deployment.yaml

```
@@ -220,7 +239,7 @@ spec:
         - name: jenkins-data
           {{- if .Values.persistence.enabled }}
           persistentVolumeClaim:
-            claimName: {{ include "common.names.fullname" . }}
+            claimName: {{ default (include "common.names.fullname" .) .Values.persistence.existingClaim }}
```

Support templates in `.Values.persistence.existingClaim`:
```suggestion
            claimName: {{ default (include "common.names.fullname" .) (tpl .Values.persistence.existingClaim $) }}
```